test/tpmclient/tpmclient.c

```diff
@@ -287,7 +286,7 @@ void Delay( UINT16 delay)
 		   __FUNCTION__, __LINE__);				\
     }									\
     									\
-    Delay(demoDelay);							\
+    Delay(0);							\
```

Comment:
At first I did but it broke something. I started to investigate but didn't want to hold up the larger effort on account of fixing this. The breakage was very strange and only broke for clang not GCC so I'm expecting that it will take a while to track down. I think you're right though: the delay isn't adding anything useful to the program and should be removed in future work.